docs/getting-started/clean-up.md

```diff
@@ -1,38 +1,37 @@
-## Clean Up
+---
+hide_next: true
+---
+# Clean Up
 
-=== "kn"
-    Delete the Service by running the command:
+We recommend that you delete the cluster used for this tutorial to free up resources
+on your local machine.
 
-    ```bash
-    kn service delete cloudevents-player
-    ```
+If you want to continue experimenting with Knative after deleting the cluster,
+you can reinstall Knative on a new cluster using the [`quickstart` plugin](quickstart-install.md) again.
```

Comment:
nit: maybe link to the section on running the plugin

```suggestion
you can reinstall Knative on a new cluster using the [`quickstart` plugin](quickstart-install.md/#run-the-knative-quickstart-plugin) again.
```
(I assume that should work for the link, but wasn't able to verify that it actually does)